trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/nestedword/operations/IsEmptyHeuristic.java

```diff
@@ -64,6 +64,7 @@
 import de.uni_freiburg.informatik.ultimate.util.CoreUtil;
 import de.uni_freiburg.informatik.ultimate.util.HashUtils;
 import de.uni_freiburg.informatik.ultimate.util.datastructures.HashedPriorityQueue;
+import de.uni_freiburg.informatik.ultimate.util.datastructures.relation.Pair;
```

Comment:
Are the changes in this class related to parallel TAR? The waypoint stuff seems witness-related?